### PR TITLE
Feature/editing readme

### PR DESCRIPTION
**Pull request by johnnystefan**
We were adding part of the steps to run the project locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+# App Xettup
+
+## Installation backend
+Lo primero que debemos es allanar las bases de nuestro entorno de desarrollo:
+
+> ¿Como lo haremos?
+
+Previamente tener instalado:
+
+- Python 3.x, [Python docs](https://www.python.org/).
```
Comment (Anfercode) [inline README.md]: La versión es python 3.8, también agregar como prerrequisito pipenv

File: README.md [hunk 2]
```diff
@@ -1 +1,54 @@
...
+# Xettup App
+
+## Installation backend
+The first thing we must do is pave the foundations of our development environment:
+
+> How will we do it?
+
+Previously have installed:
+
+- Python 3.x, [Python docs] (https://www.python.org/).
```
Comment (Anfercode) [inline README.md]: The python version is 3.8 and add pipenv to prerequirement

File: README.md [hunk 3]
```diff
@@ -1 +1,54 @@
...
+- Docker, [Docker docs] (https://docs.docker.com/engine/install/).
+- Docker Compose, [Docker docs] (https://docs.docker.com/compose/install/).
+
+Once the above is ready, we continue with:
+
+- install pipenv
```
Comment (Anfercode) [inline README.md]: Here will be the install of packages of the project, pipenv install

File: README.md [hunk 4]
```diff
@@ -1 +1,54 @@
...
+backend
+│ .docker-compose.override.yml
+│
+`` ''
+
+- Copy and paste the following into the **. Docker-compose.override.yml **
+`` ''
+version: "3.3"
+
+services:
```
Comment (Anfercode) [inline README.md:44]: Its ok, but how i run the project

File: README.md (hunk 2 above)
Comment (johnnystefan) [reply README.md]: resolved

File: README.md (hunk 1 above)
Comment (johnnystefan) [reply README.md]: resolved

File: README.md (hunk 3 above)
Comment (johnnystefan) [reply README.md]: resolved

File: README.md (hunk 4 above)
Comment (johnnystefan) [reply README.md:44]: resolved

File: README.md [hunk 5]
```diff
@@ -1 +1,65 @@
...
+    environment:
+      POSTGRES_HOST_AUTH_METHOD: ${PROTOTYPE_DB_HOST_AUTH_METHOD}
+    ports:
+      - "5432:5432"
+```
+
+- write and run the next commands lines:
+
+```bash
+make install
```
Comment (Anfercode) [inline README.md]: the command `make install` its run with the command `make start`

**Pull request merged**